### ISSUE TITLE
VoiceOver should announce "Close button" instead of "Button"

### ISSUE THREAD

**Issue by SimonBasca** [mozilla-mobile/firefox-ios#10610]
## Steps to reproduce
1. Activate VoiceOver
2. Tap on the 'x' icon from the default browser card
### Expected behavior
- VoiceOver should announces: "Close button"
### Actual behavior
- VoiceOver should announces: "Button"
### Device & build information
* Device: iPhone 12 Pro (15.4)
* Build version: 9000 (9093)
### Notes
Attachments:

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/FXIOS-4164)

**Comment by data-sync-user** [comment]
➤ Simion Basca commented:

Verifying as fix on main 9eea0a004e02ebdce2a6acd4e48b841212aba05b
